ota-plus-web/app/reactapp-new/src/components/DocsLinks/DocsLinks.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Row, Col } from 'antd';
+
+import './DocsLinks.scss';
+
+const DocsLinks: React.FC<{}> = () => {
+    return (
+        <Row>
+            <Col>
+                <div className="docslinks-title">Examples</div>
```

Comment:
Do not hardcode strings.

Reply:
+1 Also want to handle the destination URLs for the doc links in L10 file (Izumi)